src/webui/webapplication.cpp

```diff
@@ -140,6 +139,22 @@ namespace
             return QUrl(QLatin1String("http://") + hostHeader);
         return hostHeader;
     }
+
+    QString getCachingInterval(QString contentType)
+    {
+        contentType = contentType.toLower();
+
+        if (contentType.startsWith(QLatin1String("image/")))
+            return QLatin1String("private, max-age=604800");
```

Comment:
Can you add a comment with how long this equates to, or alternatively use a well-named variable.

Reply:
Done.